src/dao/ERC20DAO.js

```diff
@@ -72,18 +72,15 @@ export default class ERC20DAO extends AbstractTokenDAO {
   }
 
   async totalSupply (): BigNumber {
-    const totalSupply = await this._call('totalSupply')
-    return this.removeDecimals(totalSupply)
+    return await this._call('totalSupply')
```

Comment:
do not use `async` and `await` for awaited return. use `func(): Promise {}` for linting